index.html

```diff
@@ -457,9 +479,10 @@ <h2>
           <li>Wait until the {{Document}} of the <a>top-level browsing
           context</a> is <a>completely loaded</a>.
           </li>
-          <li>If there is already an <a>installation process</a> being
-          <a data-lt="present an install prompt">presented</a>, terminate this
-          algorithm.
+          <li>If there is already an <a data-lt=
+          "present an install prompt">install prompt being presented</a> or if
```

Comment:
note: you shouldn't need the `data-lt` here.

Reply:
This seems to be the one place we do need it (since "install prompt being presented" isn't defined as an alias, and it seems too specific to do so).